src/main_app/components/NavBar/MainMenu/styles.module.scss

```
@@ -49,3 +54,10 @@
   margin-right: 11px;
   margin-left: 4px;
 }
+
+
+.buttonTalk {display: block;
+border: antiquewhite 1px solid;
+text-align: center;
+border-radius: 4px;
+padding: 10px 15px;}
```

`antiquewhite`? Never seen that color before 😂 Can we use its Hex or RGB equivalent?

```suggestion
.buttonTalk {
  display: block;
  border: antiquewhite 1px solid;
  text-align: center;
  border-radius: 4px;
  padding: 10px 15px;
}
```